tests/test_blocks.py

```diff
@@ -1,12 +1,18 @@
 from collections import OrderedDict
 
 import pytest
-from wagtail.core.blocks import StructValue
+from wagtail.blocks import StructValue
```

Comment:
**Question:** Can this be removed?

Reply:
Nice catch, I forgot to remove it.

Reply:
Fixed in https://github.com/wagtail/wagtail-factories/pull/54/commits/d4c877d2c169a9b39b04b47506626edea4a93716